bundles/net.tourbook.export/src/net/tourbook/export/TourExporter.java

```diff
@@ -423,9 +432,11 @@ private void doExport_22_MinMax_LatLon(final VelocityContext vcContext) {
       double max_longitude = -180.0;
 
       final List<?> routes = (List<?>) vcContext.get("routes"); //$NON-NLS-1$
-      if (routes != null) {
```

Comment:
Why are some simple `null` checks being replaced with a method

Reply:
I recently discovered that [some methods](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#isNotEmpty(java.util.Collection)) can check for null and emptyness in only 1 test, which I found convenient and descriptive when reading them